apps/quickstart-integrations-monday-code/src/queue-service.js

```diff
@@ -13,14 +13,14 @@ export const produceMessage = async (message) => {
 }
 
 export const readQueueMessage = ({ body, query }) => {
-    const topicMessageSecret = process.env.MNDY_TOPIC_MESSAGES_SECRET
-    const receivedSecret = query.secret;
-    logger.info(`queue message topic Message Secret: "${topicMessageSecret}"`)
+    const envMessageSecret = process.env.MNDY_TOPIC_MESSAGES_SECRET;
+    logger.info(`expected queue secret value: ${envMessageSecret}`)
     logger.info(`queue message received body ${JSON.stringify(body)}`)
     logger.info(`queue message query params ${JSON.stringify(query)}`)
-    if (receivedSecret !== topicMessageSecret)  {
+    if (!queue.validateMessageOrigin(query.secret))  {
```

Comment:
The function signature doesn't sound intuitive to me.. "Validate Message Origin" makes me think it takes a message as parameter

How about `validateMessageSecret`?